http/watcher.go

```diff
@@ -115,10 +114,20 @@ func (w *BTCAddressWatcher) performRegisterPegIn(txHash string) error {
 	if err != nil {
 		return err
 	}
-	_, err = w.rsk.RegisterPegIn(opt, q, signature, rawTx, pmt, big.NewInt(int64(bh)))
+	err = w.rsk.RegisterPegInWithoutTx(q, signature, rawTx, pmt, big.NewInt(bh), txHash)
+	if err != nil {
+		if strings.Contains(err.Error(), "Failed to validate BTC transaction") {
```

Comment:
Let's change the LBC so that the error code is in the error message, and look for the code instead of the whole text.

Reply:
ok that will be done afterwards right?